addon/components/rdfa-editor-container.js

```diff
@@ -0,0 +1,83 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class RdfaEditorContainerComponent extends Component {
+  @tracked editor;
+  plugins = [];
+  get editorOptions() {
+    return (
+      this.args.editorOptions ?? {
+        showToggleRdfaAnnotations: Boolean(this.args.showToggleRdfaAnnotations),
+        showInsertButton: false,
+        showRdfa: true,
+        showRdfaHighlight: true,
+        showRdfaHover: true,
+        showPaper: true,
+        showSidebar: true,
```

Comment:
I think some of these can be set to false to make it more suitable for the "inline" editor use-case?